Assets/VRTK/Scripts/Interactions/VRTK_InteractTouch.cs

```diff
@@ -64,6 +66,8 @@ public class VRTK_InteractTouch : MonoBehaviour
         private VRTK_ControllerEvents.ButtonAlias originalGrabAlias;
         private VRTK_ControllerEvents.ButtonAlias originalUseAlias;
 
+        public GameObject ControllerAlias { get { return controllerEvents.ControllerAlias; } }
```

Comment:
Is this used somewhere?

Seems like this was intended to be used instead of using `controllerEvents.ControllerAlias` all the time below. I think dropping this makes the most sense to keep things simple (especially the public API).